build.Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM golang:1.8
+FROM golang:1.9-alpine
```

Comment:
@timoreimann the rc2 was released weeks ago, and the final release date was theoretically fixed weeks ago, and also https://github.com/golang/go/milestone/49 ;)